### PR TITLE
feat: add README.md template

### LINKED ISSUE #65: README.md template for Berty projects
The goal of this issue is to share ideas about the organization of our README files and potentially create an easy-to-use template

---

My initial idea of layout:
* have a generic header
  * logo
  * repo badges
  * community links (use badges?)
* body
  * short head-line that fits in a tweet
  * quick intro that can be read by a non-tech, explaining what this project is about
  * optional quick "clarification" intro for techs
  * optional how-to use (for a non-dev)
  * optional how-to dev (for dev)
  * optional how-to contribute (for non-dev)
* have a generic footer
  * maintainers section
  * contributing/community section
  * licensing section

---

Also, does it make sense to use an existing standard, if yes, which one(s) and why?
In any case, what cool ideas can we steal from those standards?

Any concerns about using smileys on sections?

Also, I think that we should write the `CODE_OF_CONDUCT.md` file, and maybe other common files only in this repository; and then use links to this repo

---

Draft document here: https://hackmd.io/28Ipo7jhQAOaZW12-FN1-g?edit

### PR DESCRIPTION
**Pull request by moul**
Fixes #65

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,22 +4,54 @@
...
+<h3 align="center">Berty is a secure peer-to-peer messaging app that works with or without internet access, cellular data or trust in the network</h3>
 
-<p align="center"><b>
-    <a href="https://berty.tech">berty.tech</a> •
-    <a href="https://twitter.com/berty">Twitter</a> •
-    <a href="https://crpt.fyi/berty-discord">Discord</a> •
-    <a href="https://github.com/berty">GitHub</a>
-</b></p>
+<p align="center">
+    <a href="https://berty.tech"><img alt="berty.tech" src="https://img.shields.io/badge/berty.tech-2845a7?logo=internet-explorer&style=flat" /></a>
```
Comment (aeddi) [inline README.md:10]: Lol

File: README.md [hunk 2]
```diff
@@ -4,22 +4,54 @@
...
+    <a href="https://twitter.com/berty"><img alt="twitter" src="https://img.shields.io/twitter/follow/berty?label=%40berty&style=flat&logo=twitter" /></a>
+</p>
 
 ---
 
 > General discussions and documentation on community practices
 
+## About
+
+Welcome to the Berty family. We should warn you: it’s a pretty big family and our home is all over the world. Wherever you are, we welcome you in.
```
Comment (aeddi) [inline README.md:22]: I'm not sure about this intro, it sounds really weird.

Comment (aeddi) [reply README.md:22]: II would have replaced this passage with something like: 
> Welcome to the Berty Project, we are an NGO that defends privacy (...)
If these topics speak to you and your values correspond to ours, we would very much like you to join our community.

File: README.md (hunk 1 above)
Comment (moul) [reply README.md:10]: :)

File: README.md (hunk 2 above)
Comment (moul) [reply README.md]: fixed :+1:

**Pull request merged**